app/src/resources/radii/index.ts

```diff
@@ -1,6 +1,13 @@
 import Radius from "@/models/radius";
 
 import radiiData from "./radii.json";
+
 const radii: Radius[] = radiiData as Radius[];
+// now this object can not be changed anymore
+Object.freeze(radii);
+
+const getDefaultRadius = (): Radius => Object.assign({}, radii[0]);
+
+export { radii, getDefaultRadius };
```

Comment:
Double exports were removed in #345. also see  #295.
We should not add it again.